mcl_3dl:
  map_frame: "map"
  accum_cloud: 2
  fake_imu: true
  lpf_step: 1.0
  num_particles: 128
  resample_var_yaw: 0.05
  resample_var_pitch: 0.1
  resample_var_roll: 0.1
  acc_var: 1.2
  likelihood:
    match_dist_min: 0.3
    match_output_dist: 0.3
obj_to_pointcloud:
  frame_id: "map"
  points_per_meter_sq: 1200.0
  downsample_grid: 0.1
pcd_to_pointcloud:
  frame_id: "map"
  latch: true
